cmd/sharded-test-server/frontproxy.go

```diff
@@ -158,13 +158,13 @@ func startFrontProxy(
 
 	go func() {
 		<-ctx.Done()
-		cmd.Process.Kill() // nolint: errcheck
+		cmd.Process.Kill() //nolint:errcheck
 	}()
 
 	terminatedCh := make(chan int, 1)
 	go func() {
 		if err := cmd.Wait(); err != nil {
-			if exitErr, ok := err.(*exec.ExitError); ok { // nolint: errorlint
+			if exitErr, ok := err.(*exec.ExitError); ok { //nolint:errorlint
```

Comment:
Why not use `errors.As` here?

Reply:
For now I didn't focus too much on rewriting the code and exploring all linters, but sure, I can enable the linter for error comparisions and fix all cases where we still use assertions or direct comparisons.